src/pcre2_stubs.c

```diff
@@ -248,8 +248,9 @@ static inline void raise_bad_pattern(int code, size_t pos)
   CAMLparam0();
   CAMLlocal1(v_msg);
   value v_arg;
-  v_msg = caml_alloc_string(128);
-  pcre2_get_error_message(code, (PCRE2_UCHAR *)String_val(v_msg), 128);
+  char msg_buf[128];
+  pcre2_get_error_message(code, (PCRE2_UCHAR *)msg_buf, (sizeof msg_buf) / (sizeof (PCRE2_UCHAR)));
+  v_msg = caml_copy_string(msg_buf);
```

Comment:
This is wrong for libpcre2-16 and -32 (since a UTF-16 or -32 character could have a zero byte; but probably unlikely come up in an error message string?). My assumption is that is fine given the dune-project relies on conf-libpcre2-8.